### DISCUSSION TITLE
What's the difference between GDDM and DDM when giving the same range of drift and fixing the other parameters?

### DISCUSSION THREAD

**Discussion by DianaY-J** [mwshinn/PyDDM#108] [Ask for help]
Hey Github folks,

I'm trying to use PYDDM to fit my data, and I tried both GDDM and DDM. It's the same data, I used the same range for drift, and fix all the other parameters to the same constant number, it's the same optimisation method 'differential evolution' as well. However, the drift rate for all the participants are slightly higher in GDDM, and the BIC is in general much better in GDDM as well. I

 like the result that GDDM performs better, but I don't get it. I knew GDDM is able to adjust the drift rate, noise, and boundary over time, but I didn't implement a varying drift rate, I gave the same range of drift for the model to estimate, and I fix all the other parameters. I would expect the same result from GDDM and DDM, but there's a significant improvement in GDDM. 

I'm curious about this result, and will really appreciate your help and thoughts. Thank you in advance!

**Comment by mwshinn** [comment]
The DDM is a special case of the GDDM.  I don't understand what you mean by fitting a DDM vs a GDDM, since in your case they are the same thing.  Fitting a model twice will not always give you the same result because the fitting method is stochastic.  It is important to check the robustness of your model fits.

**Reply by DianaY-J** [reply]
Thank you for your reply! 

I mean I would expect the same result from pic 1 and pic 2, but the actual result from these two or actually in pic 3, where you can see a significant and consistent difference between DDM and GDDM, no matter how many times I run these codes, this result stays the same. Do you perhaps know why? :)

<img width="698" alt="Screenshot 2024-07-26 at 22 02 54" src="https://github.com/user-attachments/assets/fcb4ddac-c300-4d8e-819d-c3c241fec08e">
<img width="711" alt="Screenshot 2024-07-26 at 22 03 04" src="https://github.com/user-attachments/assets/10a49abd-ca98-47f0-9ba0-864b30949642">
<img width="634" alt="Screenshot 2024-07-26 at 22 03 15" src="https://github.com/user-attachments/assets/5fab1af4-c882-4777-acaa-8bb73d381642">
